R/admiral_options.R

```diff
@@ -0,0 +1,157 @@
+admiral_options <- new.env(parent = emptyenv())
+admiral_options$subject_keys <- vars(STUDYID, USUBJID)
+# admiral_options$future_input <- vars(...) nolint
+
+######################################################
+### To enhance feature and add inputs as necessary ###
+######################################################
+
+# 1. Add additional options such as future_input as shown commented above
+# 2. Update @params with future_input in set_admiral_options roxygen documentation
+# 3. Add future_input into set_admiral_options() formals and body
+
+#' Get the value of an admiral option
+#'
+#' Get the value of an admiral option which can be modified for advanced users.
+#'
+#' @param option A character scalar of commonly used admiral function inputs.
+#'
+#'   As of now, support only available for `r enumerate(ls(admiral_options), quote_fun = dquote, conjunction = "or")`.
+#'   See `set_admiral_options()` for a description of the options.
+#'
+#' @details
+#' This function allows flexibility for function inputs that may need to be repeated
+#' multiple times in a script, such as `subject_keys`.
+#'
+#' @author Zelos Zhu
+#'
+#' @return
+#' The value of the specified option unless user changes the default
+#' using `set_admiral_options()`.
```

Comment:
I would remove "unless user changes the default using `set_admiral_options()`" because the function returns the value of the option also if it was changed.